docs/reference-guide/modules/events/pages/event-processors/index.adoc

```diff
@@ -6,38 +6,26 @@ _Event Processors_ are the components that take care of the technical aspects of
 They start a xref:messaging-concepts:unit-of-work.adoc[unit of work] and possibly a transaction.
```

Comment:
This reference will be broken as soon as we merge https://github.com/AxonFramework/AxonFramework/pull/3967, right? Or do the aliases take care of that?